hodor.gemspec

```diff
@@ -33,6 +33,12 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "chronic"
   spec.add_runtime_dependency "ox"
 
+  # Gems needed for managing edn configs (Secret handling)
+
+  spec.add_runtime_dependency 'edn'
+  spec.add_runtime_dependency 'aws-sdk'
+  spec.add_runtime_dependency 'activesupport'
```

Comment:
As we discussed, I'm fine with including activesupport, as long as it doesn't pull in a huge number of dependencies of its own.